Geometry/TrackerNumberingBuilder/bin/stubs/GeometricDetLoader.cc

```diff
@@ -37,7 +38,7 @@ void GeometricDetLoader::beginRun(edm::Run const& /* iEvent */, edm::EventSetup
     std::cout << "PoolDBOutputService unavailable" << std::endl;
```

Comment:
When possible use `LogVerbatim`. see https://github.com/cms-sw/cmssw/issues/29610

Reply:
There are a load of couts in all these files - maybe changing them over to `LogVerbatim` should be decoupled from this PR?

Reply:
@adewit Yes, the `cout` fix can be done later.